src/api/postFile.ts

```diff
@@ -1,18 +1,20 @@
 import axios from 'axios';
 import { url } from '../shared/variable.shared';
+import { useUserStore } from 'src/stores/user';
 
-
-const changeFileToFormData = (file: File): FormData => {
+const changeFileToFormData = (file: File)=> {
     const formData = new FormData();
+    // const userStore = useUserStore();
+    // formData.append('email', userStore.getUser.email ?? '');
     formData.append('file', file, file.name);
     return formData
 }
 
-
 const sendFile = async (file: File): Promise<boolean> => {
     const formData = changeFileToFormData(file);
+    const userStore = useUserStore();
     try{
-        await axios.post(url+'/files', formData, { headers: {
+        await axios.post(url+'/files?email='+userStore.getUser.email ?? '', formData, { headers: {
```

Comment:
Zmieniłem na query param, żeby Piotr mógł odebrać ten plik.  Wiem, że to nie jest dobre rozwiązanie, ale i tak będziemy to przerabiać na userID, który Piotr pewnie niedługo zrobi.

Reply:
takie cos by nie zadzialalo?
axios.post(url+'/files', formData, {params: {email}}, { headers: {}}
https://stackoverflow.com/questions/53501185/how-to-post-query-parameters-with-axios

Reply:
Jeśli wolicie przez id to mozesz nawet teraz zrobic, mowilem ze mi to nie problem przerobic xd

Reply:
A to nie tak będzie?
`axios.post(url+'/files', formData, {params: {email}, headers: {}}`

Reply:
W taki sposób małpę na procenty podmienia: 
![image](https://user-images.githubusercontent.com/78599029/211218788-36cee93b-689c-4f79-9168-f210bb4cc3b8.png)

Zapytanie:
```
axios.post(url+'/files', formData, { params: {email: userStore.getUser.email ?? ''}, headers: {
            'Content-Type': `multipart/form-data`
        } })
```

Ale dobrze jest odczytywany na backendzie:
![image](https://user-images.githubusercontent.com/78599029/211218838-87756d39-b86d-4a90-8366-405df45d46c5.png)

**Zostawić?**

Reply:
query params są kodowane więc to normalne zachowanie

Reply:
Pytanie czy nie lepiej wysyłać email w body, ale to juz inny temat